frontend/views/pages/DesignSystem.vue

```diff
@@ -620,81 +628,63 @@ page(
-              |    input.input(type='text')
-          td
-            .field
-              label.label This is a text input
-              input.input(type='text' placeholder='Placeholder')
-
-            .field
-              label.label This is a text input
-              input.input(type='text' placeholder='Placeholder')
-
         tr
           td
-            pre
-              | .select-wrapper
-              |   input.input(type='text')
-              |   select
-              |     option USD
-              |     option BTC
-              |     option EUR
-          td
-            .select-wrapper
-              input.input(type='text' placeholder='New amount')
-              select
-                option USD
-                option BTC
-                option EUR
-
+            h3 Combination
         tr
           td
             pre
-              | .input-combo
-              |    input.input(type='text')
-              |    label USD
-          td
-            .input-combo
-              input.input(type='text' placeholder='New amount')
-              .suffix USD
-
+              | label.field
+              |   .label Mincome
+              |   .select-wrapper.error
+              |     input.input
+              |     select
+              |       option USD
+          td
+            label.field
+              .label Mincome
+              .select-wrapper(:class='{ error: ephemeral.forms.hasError }')
+                input.input(placeholder='Amount')
+                select
+                  option USD
+                  option BTC
+                  option EUR
+              span.error(v-if='ephemeral.forms.hasError') Something went wrong
         tr
           td
             pre
-              | .input-combo
-              |    input.input(type='text')
-              |    button.is-icon
-              |       i.icon-eye
-          td
-            .input-combo
-              input.input(type='text' placeholder='Placeholder')
-              button.is-icon
-                i.icon-eye
-
+              | label.field
+              |   .label New Amount
+              |   .input-combo.error
+              |     input.input
+              |     .suffix USD
+          td
+            label.field
+              .label New Amount
+              .input-combo(:class='{ error: ephemeral.forms.hasError }')
+                input.input(type='text' placeholder='New amount')
+                .suffix USD
+              span.error(v-if='ephemeral.forms.hasError') Something went wrong
         tr
           td
+            | A11Y: Add #[code aria-label] to button.
             pre
-              | .input-combo
-              |    input.input(type='text')
-              |    button.is-icon
+              | label.field
+              |   .label Password
+              |   .input-combo.error
+              |     input.input(type='text')
+              |     button.is-icon(
+              |       aria-label='Show password'
```

Comment:
Remember, if you're giving examples for how something must be done, than anywhere there are user-facing strings they must be localized. So here `:aria-label:='L("Show password")'`, and elsewhere too, make sure the appropriate `L` or `i18n` are added

Reply:
This is the Design System page. I thought we had agreed about not being needed to use i18n here.

Reply:
This PR isn't about fixing the design system page, so let's ignore it for now, but my understanding was that the design system page wouldn't use `i18n` itself in its code, but when it's showing code examples (as with the use of a `pre` tag), it should inform users how to properly write their code, so that means using `i18n`.

I will add a note to #419 to make it simple to ignore files, so that it doesn't parse the design page for strings.

Reply:
Hum... I disagree. I think we should keep the examples as simple as possible and focus on the component itself only. i18n is an extra, that can be added later in its own section when we solve #607